apps/designer/app/designer/features/props-panel/use-props-logic.ts

```diff
@@ -117,7 +117,7 @@ export const usePropsLogic = ({
       const isPropRequired = draft[index].required;
       switch (field) {
         case "prop":
-          if (!isPropRequired) {
+          if (isPropRequired !== true) {
```

Comment:
could have been === false 😅

Reply:
`!== true` is needed to handle `undefined`. isPropRequired is `boolean | undefined`